snap/snapcraft.yaml

```diff
@@ -20,13 +20,13 @@ adopt-info: certbot
 
 apps:
   certbot:
-    command: certbot.wrapper
+    command: $SNAP/bin/python3 $SNAP/bin/certbot
```

Comment:
I wasn't able to build this with `snapcraft --use-lxd`:

    ....
    Staging certbot
    Staging shared-metadata
    Priming certbot
    Priming shared-metadata
    Failed to generate snap metadata: The specified command '$SNAP/bin/python3 $SNAP/bin/certbot' defined in the app 'certbot' is not executable.
    Run the same command again with --debug to shell into the environment if you wish to introspect this failure.

Reply:
I may have been to much zealous by adding `$SNAP`. I am retrying, maybe I did not cleaned properly before rebuilding.

Reply:
Ok, super weird. The executable in the command (`bin/python3`) must be relative to the snap installation path, but is not executed with the current working directory set to it, because the script argument (`bin/certbot`) is not found if also set relatively. It must be absolute. I suppose snapd just appends in some way the path to the executable dynamically.

Finally it is working with `bin/python3 $SNAP/bin/certbot`.